Util/ClassExtractor.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace A5sys\TypeScriptGeneratorBundle\Util;
+
+/**
+ * Copied from by Symfony's ClassMapGenerator which is now deprecated
+ * https://github.com/symfony/class-loader/blob/3.3/ClassMapGenerator.php
+ */
+class ClassExtractor
+{
+    /**
+     * Iterate over all files in the given directory searching for classes.
+     *
+     * @param \Iterator|string $dir The directory to search in or an iterator
+     *
+     * @return array A class map array
+     */
+    public static function createMap($dir): array
+    {
+        if (is_string($dir)) {
+            $dir = new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($dir));
+        }
+
+        $map = array();
+
+        foreach ($dir as $file) {
+            if (!$file->isFile()) {
+                continue;
+            }
+
+            $path = $file->getRealPath() ?: $file->getPathname();
+
+            if ('php' !== pathinfo($path, PATHINFO_EXTENSION)) {
+                continue;
+            }
+
+            $classes = static::findClasses($path);
+
+            if (\PHP_VERSION_ID >= 70000) {
```

Comment:
in a constant pls

Reply:
Since this bundle requires PHP 7.1 I can even delete this check 👍 (Once again, it's just a copy from Symfony's repository)